getting-started-guides/on-prem-cluster/standalone-scala.md

```diff
@@ -66,6 +67,48 @@ ${SPARK_HOME}/sbin/start-slave.sh ${SPARK_MASTER} -c ${SPARK_CORES_PER_WORKER}
 
 Note that in this example the Master and Worker processes are both running on the same host. This is not a requirement, as long as all hosts that are used to run the Spark app have access to the dataset.
 
+<span id="etl">Launch Mortgage ETL</span>
+---------------------------
+If user wants to use a larger size dataset other than the default one, we provide an ETL app to process raw Mortgage data.
+
+Variables required to run spark-submit command:
+```
+# path to xgboost4j_spark/libs
+export LIBS_PATH=/home/xgboost4j_spark/lib
+
+# additional jars for XGBoost4J example, use corresponding cudf jar accoding to your cuda runtime version.
+export SPARK_JARS=${LIBS_PATH}/cudf-0.14-cuda10-2.jar
```

Comment:
CUDF_JAR ??

Reply:
Update, since jars are moved to SPARK_HOME/jars. not extra --jars required here.